privacy-snapshot.json

```diff
@@ -30,6 +30,7 @@
   "etherscan.io",
   "execution.metamask.io",
   "fonts.gstatic.com",
+  "foo.io",
```

Comment:
?

Reply:
this comes from an example param in the api-specs

Reply:
Why does it show up in the privacy snapshot?

Reply:
because the api-specs CI job uses the [example param from the api-specs package](https://github.com/MetaMask/api-specs/blob/1f763929bbe781d6f2abefee86fd11a829595fe5/openrpc.yaml#L461) when it runs. I'm not sure what specific set of changes in this PR cause this to trigger the privacy snapshot now. I can check again later

Reply:
Okay, good point. I don't think there is a reason we need to actually make this request. I will see if I can mock it out.

Reply:
@davidmurdoch Okay, I was not able to figure out how to avoid adding `foo.io` to the privacy snapshot, however, I did mock out this request. I confirmed that it was mocked by throwing an error in the response function.

[cf5e8e3](https://github.com/MetaMask/metamask-extension/pull/27782/commits/cf5e8e3d0fa4881c6017c31674398d3372c1fec9) [9714b30](https://github.com/MetaMask/metamask-extension/pull/27782/commits/9714b30b65b1f7633f3a5fb4f5d9ca9ee3241d4d)

I will update `api-specs` in another PR to use a different domain.

Does that address your concerns?